djitellopy/tello.py

```diff
@@ -32,7 +32,7 @@ class Tello:
 
     # Video stream, server socket
     VS_UDP_IP = '0.0.0.0'
-    VS_UDP_PORT = '11111'
```

Comment:
@damiafuentes Here's the requested changes.
I trashed all my propellers, so I can only test my `python examples/simple.py` variant when backup arrives in the mail.